<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportOptions</class>
 <widget class="QWidget" name="ExportOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="optionsPlaceHolderLayout"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Thumbnail:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QLineEdit" name="thumbnailLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="screenshotButton">
       <property name="text">
        <string>Screenshot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="commentTextEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="publishButton">
     <property name="text">
      <string>Publish!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="publishMessageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExportOptions</receiver>
   <slot>setThumbnailFilename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>screenshotButton</sender>
   <signal>clicked()</signal>
   <receiver>ExportOptions</receiver>
   <slot>takeScreenshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setThumbnailFilename()</slot>
  <slot>takeScreenshot()</slot>
 </slots>
</ui>
